src/containers/ProjectDashboardContainer.jsx

```diff
@@ -43,9 +44,11 @@ class ProjectDashboardContainer extends Component {
   }
 
   handleSelect({ option }) {
+    const { actions } = this.props;
     this.setState({
       option,
     });
+    actions.createTranslation('project_contents', option.value);
```

Comment:
I'm not sure. When does the cloning/creating of the resource happen then? Panoptes doesn't support directly editing the `language` param of, e.g., `project_contents`. We need  to create a new resource of that type with a different value for `language`.